spikingjelly/activation_based/cuda_kernel/spike_op.py

```
@@ -9,22 +10,26 @@
 from torch.types import _int, _size
 from torch.utils.cpp_extension import load_inline
 
+
 from . import tensor_cache
 
 try:
     import cupy
 except BaseException as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `BaseException`

(BLE001)